OWML.ModHelper.Input/BindingChangeListener.cs

```diff
@@ -11,13 +11,13 @@ public class BindingChangeListener : MonoBehaviour
         internal void Initialize(ModInputHandler inputHandler, IModEvents events)
         {
             _inputHandler = inputHandler;
-            events.Subscribe<TitleScreenManager>(Events.AfterStart);
+            events.Subscribe<TitleScreenManager>(Common.Events.AfterStart);
```

Comment:
Don't think you need `Common.`

Reply:
It thinks that 'Events' is namespace instead... Kinda your fault :p